src/Field/WfsSearchInput/WfsSearchInput.jsx

```diff
@@ -0,0 +1,442 @@
+ * so most of the WFS specific options work like document in the corresponding
+ * API-docs: https://openlayers.org/en/latest/apidoc/module-ol_format_WFS.html
+ *
+ * @class WfsSearchInput
+ * @extends React.Component
+ */
+export class WfsSearchInput extends React.Component {
+
+  /**
+   * The className added to this component.
+   * @type {String}
+   * @private
+   */
+  className = `${CSS_PREFIX}wfssearchinput`
+
+  static propTypes = {
+    /**
+     * An optional CSS class which should be added.
+     * @type {String}
+     */
+    className: PropTypes.string,
+    /**
+     * The base URL. Please make sure that the WFS-Server supports CORS.
+     * @type {String}
+     */
+    baseUrl: PropTypes.string.isRequired,
+    /**
+     * An object mapping feature types to an array of attributes that should be searched through.
+     * @type {Object}
+     */
+    searchAttributes: PropTypes.object.isRequired,
+    /**
+     * A nested object mapping feature types to an object of attribute details,
+     * which are also mapped by search attribute name.
+     *
+     * Example:
+     *   ```
+     *   attributeDetails: {
+     *    featType1: {
+     *      attr1: {
+     *        matchCase: true,
+     *        type: 'number',
+     *        exactSearch: false
+     *      },
+     *      attr2: {
+     *        matchCase: false,
+     *        type: 'string',
+     *        exactSearch: true
+     *      }
+     *    },
+     *    featType2: {...}
+     *   }
+     *   ```
+     * @type {Object}
+     */
+    attributeDetails: PropTypes.objectOf(
+      PropTypes.objectOf(
+        PropTypes.shape({
+          matchCase: PropTypes.bool,
+          type: PropTypes.string,
+          exactSearch: PropTypes.bool
+        })
+      )
+    ),
+    /**
+     * The namespace URI used for features.
+     * @type {String}
+     */
+    featureNS: PropTypes.string,
+    /**
+     * The prefix for the feature namespace.
+     * @type {String}
+     */
+    featurePrefix: PropTypes.string,
+    /**
+     * The feature type names. Required.
+     * @type {String[]}
+     */
+    featureTypes: PropTypes.arrayOf(PropTypes.string).isRequired,
+    /**
+     * SRS name. No srsName attribute will be set on geometries when this is not
+     * provided.
+     * @type {string}
+     */
+    srsName: PropTypes.string,
+    /**
+     * The output format of the response.
+     * @type {string}
+     */
+    outputFormat: PropTypes.string,
+    /**
+     * Maximum number of features to fetch.
+     * @type {number}
+     */
+    maxFeatures: PropTypes.number,
+    /**
+     * Geometry name to use in a BBOX filter.
+     * @type {string}
+     */
+    geometryName: PropTypes.string,
+    /**
+     * Optional list of property names to serialize.
+     * @type {String[]}
+     */
+    propertyNames: PropTypes.arrayOf(PropTypes.string),
+    /**
+     * Filter condition. See https://openlayers.org/en/latest/apidoc/ol.format.filter.html
```

Comment:
Link is outdated. Please make use of [https://openlayers.org/en/latest/apidoc/module-ol_format_filter.html](https://openlayers.org/en/latest/apidoc/module-ol_format_filter.html).